SPECS/cert-manager/cert-manager.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Automatically provision and manage TLS certificates in Kubernetes
 Name:           cert-manager
-Version:        1.11.2
-Release:        8%{?dist}
+Version:        1.14.4
```

Comment:
This looks like a big version jump with potential breaking changes. Is it possible to use a patch instead of the upgrade? Or were you able to verify the upgrade is safe to take in?